setup.py

```diff
@@ -36,6 +38,7 @@
           'psutil',
           'pytz',
           'pyyaml',
+          'python-toml',
```

Comment:
Hu weird, `python-toml` is a debian pkg, I'm using `toml` from pypi, that's a mistake?

Reply:
Aha, you mean https://pypi.org/project/toml/? I see that is done now in https://github.com/YunoHost/moulinette/commit/55515cb065f43d2bb7cee77504447f2b8924703a :+1: I'll drop this commit.

Reply:
Yup, added it without realising that you did it and saw it conflicted on this PR :x